doc/admin-guide/files/records.config.en.rst

```diff
@@ -2422,6 +2422,20 @@ DNS
    in DNS Injection attacks), particularly in forward or transparent proxies, but
    requires that the resolver populates the queries section of the response properly.
 
+.. ts:cv:: CONFIG proxy.config.dns.connection.mode INT 0
```

Comment:
Bikeshedding, but I think this should be named proxy.config.dns.connection_mode, to be consistent with currently naming conventions in this namespace.